ui/components/forms/flavors/select/index.scss

```diff
@@ -22,3 +22,35 @@
     }
   }
 }
+
+#{$css-prefix}form-element__select-container {
```

Comment:
D'oh! Yeah I probably need those `.`s haha.

For the second comment, wouldn't `__` technically mean the container would be a child of the `.slds-select` class? This wouldn't be the case, as the container would wrap the select itself. Please advise.